src/templates/case-study/style.js

```diff
@@ -0,0 +1,171 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+export default makeStyles(theme => ({
+  container: {
+    padding: theme.spacing(5, 0, 10, 0),
```

Comment:
The container is a bit too wide compared to the design - I played around with it a bit and looks like this is about right - see what you think!
```suggestion
    padding: theme.spacing(5, 5, 10, 5),
```

Reply:
done